qa/L0_config_search/config_generator.py

```diff
@@ -113,77 +126,9 @@ def _get_sweep_configs(profile_models):
     model_config['total_models_remote'] = 1
     sweep_configs.append(model_config)
 
-    model_config = {
```

Comment:
Why do you remove the test? Each of them is covering one corner case in the L0_config_search. I don't think we should remove tests to make them shorter.

Reply:
When the run config search is disabled, the code path only forks based on launch mode, which is covered in test.sh. I think those tests were redundant. I only removed tests where the run config search was disabled.